src/apps/schedule/api/schedule.py

```diff
@@ -43,6 +44,10 @@ async def schedule_create(
 ) -> Response[PublicEvent]:
     """Create a new event for an applet."""
     applet_service = AppletService(session, user.id)
+
+    if schema.start_time == schema.end_time:
+        raise StartEndTimeEqualError
+
```

Comment:
Please move this validation to the request model validator raising ValueError

Reply:
fixed

Reply:
fixed